__tests__/Map.ts

```diff
@@ -1,27 +1,41 @@
-import { is, List, Map, Range, Record, Seq } from 'immutable';
+import { is, List, Map as ImmutableMap, Range, Record, Seq } from 'immutable';
```

Comment:
Instead of having to convert this whole test file to use an alias, how do you feel about just making a new test file for Map to IMap?

Reply:
I did not knew how to call global constructor, I think the best way is to call global.Map as you did in another PR